src/Spiffy.Monitoring/EventContext.cs

```diff
@@ -76,34 +76,20 @@ Assembly AssemblyFor<T>()
 
         readonly Dictionary<string, object> _values = new Dictionary<string, object>();
         readonly Dictionary<string, uint> _counts = new Dictionary<string, uint>();
-        readonly Dictionary<string, AutoTimer> _timers = new Dictionary<string, AutoTimer>();
 
         readonly object _valuesSyncObject = new object();
         readonly object _countsSyncObject = new object();
-        readonly object _timersSyncObject = new object();
         
         readonly DateTime _timestamp;
         readonly AutoTimer _timer = new AutoTimer();
 
         public IDisposable Time(string key)
         {
-            AutoTimer timer;
-            lock (_timersSyncObject)
-            {
-                if (_timers.ContainsKey(key))
-                {
-                    timer = _timers[key];
-                    timer.Resume();
-                }
-                else
-                {
-                    timer = _timers[key] = new AutoTimer();
-                }
-            }
-
-            return timer;
```

Comment:
glad to see this code go away.  the custom threading code existed because this library used to target .NET 3.5 (and the concurrent namespace wasn't introduced until 4.0).

I should probably update all the code to use modern concurrent collections (given this library now targets netstandard only)

Reply:
Are there tests around the concurrency aspect here? If not, I'm curious what the new vs old behavior is around concurrent `Time` calls (with the same key). e.g. in the old one, `Resume` was called inside the lock, but in new implementation its outside any locks.

Reply:
Added concurrency tests [here](https://github.com/chris-peterson/spiffy/pull/25/commits/ca211a1360a7c16df67aa0044a194b9c565e47b8)